### ISSUE TITLE
PAL requires run-as administrator to manipulate merging templates

### ISSUE THREAD

**Issue by jeffstokes72** [clinthuffman/PAL#15]
Seems we should be able to do this even though the template xml files are stored in program files. Another fix might be to store the actual xmls in a user profile store?

**Comment by jeffstokes72** [comment]
![2018-03-07_11-54-06](https://user-images.githubusercontent.com/9102841/37105880-460bd646-21fe-11e8-811d-b14317d35f11.png)

**Comment by clinthuffman** [comment]
Correct. I find it better just to unzip PAL to the Desktop folder and use it from there and not compiling a setup MSI anymore.

**Comment by clinthuffman** [comment]
Which folder do you recommend for PAL to be installed to by default? %USERPROFILE%\Documents or is there a more appropriate place?

**Comment by jeffstokes72** [comment]
Well, actually. If you want, just ditch the MSI mode completely and do the flat files like you said on Sept 2nd I think?

If not, maybe Public\Documents is a good spot?
